specification/search/resource-manager/Microsoft.Search/preview/2025-02-01-preview/examples/SearchCreateOrUpdateServiceAuthOptions.json

```diff
@@ -60,7 +60,16 @@
               "aadAuthFailureMode": "http401WithBearerChallenge"
             }
           },
-          "disabledDataExfiltrationOptions": []
+          "disabledDataExfiltrationOptions": [],
+          "upgradeAvailable": true,
+          "systemData": {
+            "createdBy": "My e-commerce app",
+            "createdByType": "Application",
+            "createdAt": "2024-01-01T00:00:00Z",
+            "lastModifiedBy": "fakeuser@contoso.com",
```

Comment:
Assuming the "lastModified" properties are updated any time service metadata is changed by the customer, we will still need to include serviceUpgradedAt to know the version of the service was last changed. This is how we'll know the current version of the service and whether there is an upgrade available.